eventsources/sources/gitlab/start.go

```diff
@@ -216,26 +241,37 @@ func (el *EventListener) StartListening(ctx context.Context, dispatch func([]byt
 			return fmt.Errorf("failed to initialize client, %w", err)
 		}
 
-		getHook := func(hooks []*gitlab.ProjectHook, url string) *gitlab.ProjectHook {
-			for _, h := range hooks {
-				if h.URL != url {
+		router.hookIDs = make(map[string]int)
```

Comment:
Is the hook id unique, no matter if it's a project hook, or a group hook?

Reply:
Don't have to consider whether the hook id is unique between project hook and group hook, Because they are different Gitlab API and `router.hookIDs` key.

Reply:
My question is, if it's not unique across groups and projects, will using one `router.hookIDs` be a problem?

Reply:
I see what you mean now. How do you think to add another map in `router` to save group level hook ids？

Reply:
That sounds like a plan.